cinema_service/settings.py

```diff
@@ -137,3 +139,9 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+REST_FRAMEWORK = {
+    "DEFAULT_PERMISSION_CLASSES": [
+        "rest_framework.permissions.AllowAny",
+    ]
+}
```

Comment:
The `DEFAULT_PERMISSION_CLASSES` is set to `AllowAny`, which allows unrestricted access to all API endpoints. Consider using more restrictive permissions for production to secure your API.